spec/views/rooms/show.html.erb_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe "rooms/show", type: :view do
+  before do
+    @room = FactoryBot.create :room
+    # @building = FactoryBot.create :building
+  end
+
+  it "renders attributes in <p>" do
```

Comment:
Was genau bedeutet "in <p>"?

Reply:
HTML-Tag für Paragraph. Gemeint sind die Attribute in den Paragraphen